tests/wake-format/basic/stdout

```diff
@@ -95,27 +106,32 @@ def name Unit =
     # wake-format off
     def other = here
     def other = @here
+
     another
 
 def name Unit =
     def other = @here
+
     # wake-format off
     def other = here
     def other = @here
+
     another
 
 def name Unit =
     def other = @here
+
     # wake-format off
     def other = here
-
     def other = @here
```

Comment:
Is this due to the `# wake-format off` or to both of them being simple defs?  I'm expecting the second, but I'm curious if disabling the formatter would have an effect either way.  Also, weren't we planning on using the formatter to switch over to `@here`?

Reply:
> Is this due to the # wake-format off or to both of them being simple defs? 

Simple defs, the format off only applies to `def other = here`, you need to put it at the top level if you want to disable the entire function

>  Also, weren't we planning on using the formatter to switch over to @here?

Yep, but we have to respect format off even for refactors. In this case we'd have to fix the (likely few) cases manually before removing `here`

Reply:
> > Also, weren't we planning on using the formatter to switch over to @here?
> 
> Yep, but we have to respect format off even for refactors. In this case we'd have to fix the (likely few) cases manually before removing `here`

I completely missed the fact that was falling under the format-off switch -- saw it to ask about the line (my question was whether format-off for one or the other `def` would affect removing the linebreak between them or whether that's entirely controlled by the parent block), but then promply forgot it for the `here`.